src/karmen_backend/Pipfile

```diff
@@ -27,6 +27,9 @@ flask-executor = "*"
 sentry-sdk = {extras = ["flask"],version = "==0.14.3"}
 argh = "*"
 boto3 = "*"
+swagger-ui = "*"
```

Comment:
I suggest to use a certain version due to avoid a possible misfunction in a future versions.

Reply:
I suggest dropping swagger ui altogether and instead host [Redocly](https://github.com/Redocly/redoc) elswhere separately. It's a lot more readable and elegant.